toolkit/docs/building/prerequisites-mariner.md

```diff
@@ -1,18 +1,18 @@
 
 # Build Requirements on `Mariner`
 
-## Requirements were validated on `Mariner 1.0`
+## Requirements were validated on `Mariner 2.0`
 
-This works regardless of installing `core` or `full` version of mariner.
-If you installed `full` version of mariner, some of these packages are already installed, so less packages will be downloaded.
-
-Requirements for building images with a toolkit:
+This page lists host machine requirements for building with the CBL-Mariner toolkit. They cover building the toolchain, packages, and images on a Mariner host.
 
 ```bash
 # Install required dependencies.
-sudo dnf -y install git make tar rpm-build gcc glibc-devel binutils \
-kernel-headers wget curl rpm qemu-img golang cdrkit python2 bison \
-gawk parted dosfstools pigz moby-engine moby-cli
+sudo tdnf -y install binutils cdrkit curl dosfstools gawk glibc-devel \
```

Comment:
**[NOT A BLOCKER]**

This change made me think about re-arranging such instructions a bit. What do you think about breaking it into multiple lines, so that every line contains only one package? It would make tracking changes to these files a lot easier. Similarly, reviews would probably be simpler as well.